test/KillbillTest.php

```diff
@@ -32,8 +32,14 @@ class KillbillTest extends \PHPUnit_Framework_TestCase
 
     public function setUp()
     {
+        Client::$mockManager = new MockManager();
+        $externalKey = uniqid();
+        if (getenv('ENV') === 'local' || getenv('RECORD_REQUESTS') == '1') {
```

Comment:
Could this logic be extracted in a utility function, since it's needed in various places?